types/rdf-ext/lib/Dataset.d.ts

```diff
@@ -1,16 +1,11 @@
-import { Dataset, Stream, Quad } from 'rdf-js';
+import Dataset = require('rdf-dataset-indexed/dataset');
```

Comment:
Could be

```suggestion
import Dataset from 'rdf-dataset-indexed/dataset';
```

?

Reply:
No because it require `esModulesInterop = true` which is not used in any of the packages